src/Webfactory/Slimdump/SlimdumpCommand.php

```diff
@@ -59,15 +67,44 @@ private function connect($dsn)
         }
     }
 
+    /**
+     * @return int
+     */
+    protected function reformatBufferSize($size)
```

Comment:
To me, `parseBufferSize` seems to be a better name.

Reply:
Good idea, I will fix that.